MANIFEST.in

```diff
@@ -1,4 +1,6 @@
-include *.py MANIFEST.in
-global-include *.txt *.rst *.ini *.yml *.cfg *.crt *.key
-global-exclude .gitignore
+graft aiosmtpd
+global-include *.txt *.rst *.ini *.yml *.cfg *.crt *.key LICENSE
+global-exclude *.gitignore *.py[oc]
 prune build
+prune .tox
+prune .venv
```

Comment:
If we no longer use `global-include`, do we still need these `prune` directives?

Reply:
Yes. If the `*.txt` includes do not drag in .tox or .venv we can drop this.